packages/h5wasm/src/h5wasm-api.ts

```diff
@@ -101,7 +108,9 @@ export class H5WasmApi extends ProviderApi {
       }
 
       const children = h5wEntity.keys().map((name) => {
-        const h5wChild = h5wEntity.get(name);
+        const h5wChild = h5wEntity.get(name) || undefined;
+        assertDefined(h5wChild);
```

Comment:
Not bothering with a user-friendly error message here, since if the child doesn't exist, something's wrong with h5wasm, not with the HDF5 file.